lib/quest/api.rb

```diff
@@ -28,40 +28,40 @@ def active_quest_status
     end
 
     get '/status/examples' do
-      active_quest_status[:examples].to_json
+      active_quest_status['examples'].to_json
     end
 
     get '/status/examples/:number' do
-      active_quest_status[:examples][params[:number].to_i - 1].to_json
+      active_quest_status['examples'][params[:number].to_i - 1].to_json
```

Comment:
probably a good idea to standardize on either symbols or strings, and either one-based or zero-based indexing.